redact_sample.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import csv
+import logging
+import os
+import sys
+from typing import Iterable
+
+from generate_sample_file import SampleGenerator
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+SampleGen = SampleGenerator()
+SampleGen.ESTAB_TYPES = ['HALL OF RESIDENCE', 'CARE HOME', 'HOSPITAL', 'HOSPICE', 'MENTAL HEALTH HOSPITAL',
```

Comment:
I would prefer that this was put in an 'NONE_SENSITIVE_ESTAB_TYPES' constant and then set against the SampleGen.  This would hopefully explain what's going on.

Reply:
in fact, why cant we subtract the sensitive list from the other list, to get the non-sensitive types: let python do the heavy lifting